.circleci/config.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - run:
           name: Helm Lint Check (Enterprise)
           command: helm lint --set enterprise.enabled=true --set enterprise.license=123asd456fake
+      - run:
+          name: Helm Lint Check (st2Repository)
+          command: helm lint --set image.st2Repository="docker.example.com/st2modified"
```

Comment:
It's a bit excessive to have a dedicated lint check for this small thing.
I think this would better fit into #28 Unit Tests which (we don't have yet).

Reply:
removed